powershell/autorest-configuration.md

```diff
@@ -137,9 +137,12 @@ pipeline:
   psnamer-v2:
     input: csnamer-v2
 
-  add-azure-completers-v2:
+  modifiers-v2:
     input: psnamer-v2
 
+  add-azure-completers-v2:
+    input: modifiers-v2
+
   llcsharp-v2:
     input: add-azure-completers-v2
```

Comment:
Notice an issue in preview change. llcsharp-v2 should rely on modifiers-v2 instead of add-azure-completers-v2